pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -246,9 +246,6 @@ protected void updateTopicPolicyByNamespacePolicy(Policies namespacePolicies) {
         if (log.isDebugEnabled()) {
             log.debug("[{}]updateTopicPolicyByNamespacePolicy,data={}", topic, namespacePolicies);
         }
-        if (namespacePolicies.deleted) {
```

Comment:
Remove `namespacePolicies.deleted` logic to ensure the new topic will not be deleted by `PersistenTopic#checkReplication`.

Reply:
with https://github.com/apache/pulsar/pull/19097 we will be sure that "namespacePolicies.deleted" is authoritative.
if we are deleting the namespace then we should do nothing

cc @dlg99 @aymkhalil

Reply:
Hi @eolivelli 
I don't understand what you mean.

Will removing this test affect anything?

Reply:
@eolivelli 

If I don't remove this check, the creating topic in the deleted namespace will automatically be forcefully deleted by `checkReplication` method because the `replicationClusters` is null. See the logic details:

https://github.com/apache/pulsar/blob/17c58a539315cc4ea39655d4328c5caf55f87d3d/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java#L1468-L1479

Reply:
I have checked the code again.
We can keep this change, it is harmless, the topic is deemed to be deleted, we can allow the policy to be updated